### ISSUE TITLE
Audio only is misssing

### ISSUE THREAD

**Issue by tristan-k** [tastelessjolt/youtube-dl-android#17]
I miss downloading only the audio as a format option.

**Comment by tastelessjolt** [comment]
I believe it is still there in the list of formats, you can scroll down to the last of the list where you can find it.

**Comment by tristan-k** [comment]
Not for me. There is just 'Video Only'.

![](https://i.imgur.com/4uD2sAE.png)

**Comment by tastelessjolt** [comment]
Can you post the link of the video that you're trying to download?

**Comment by tristan-k** [comment]
Sure. [Here](https://www.youtube.com/watch?v=a772CHFvX8o) it is.

**Comment by tastelessjolt** [comment]
It looks like even `youtube-dl` can't find the audio file link. 

![image](https://user-images.githubusercontent.com/16670547/52058926-78931680-258e-11e9-8110-67ebbe90032b.png)

You may raise an issue there since this is just a one-to-one copy of that app.

Edit: Looks like I'm wrong, there is an audio file link.

**Comment by tristan-k** [comment]
NewPipe downloads from the channel just fine.

![](https://i.imgur.com/29rJPtc.png)

**Comment by tigefa4u** [comment]
[ASK] where is the .MP3 🤔

**Comment by tastelessjolt** [comment]
Yeah, this is a bug. 
In the list of formats, it doesn't show just the last format because of layout issues, and it happens to be that most "Audio only" formats are at the end of the formats list.

**Comment by next2non** [comment]
Yes please could you add the mp3 option to download. Also video quality along with audio could have more options. Currently only 360. I would like to download for example 1080 video+audio but don’t see that option, only 1080 video without audio
